featuretools/primitives/standard/aggregation/max_consecutive_zeros.py

```diff
@@ -23,6 +23,10 @@ class MaxConsecutiveZeros(AggregationPrimitive):
     """
 
     name = "max_consecutive_zeros"
+    input_types = [
+        [ColumnSchema(logical_type=Integer)],
+        [ColumnSchema(logical_type=Double)],
+    ]
     input_types = [ColumnSchema(semantic_tags={"numeric"})]
```

Comment:
Input types is defined twice on L26 and L30.

Reply:
Removed